core/lib/instrument_api.h

```diff
@@ -4553,6 +4553,22 @@ DR_API
 bool
 dr_raw_tls_cfree(uint offset, uint num_slots);
 
+/**
+ * Provides raw access to offset \p offset from TLS base \p tls_register
+ * and writes value \p value.
```

Comment:
This should be referenced in the dr_raw_tls_calloc docs like the other raw TLS routines.

Reply:
Done.